docs/scenarios/parametric.md

```diff
@@ -97,9 +92,36 @@ TEST_LIBRARY=dotnet ./run.sh PARAMETRIC -k test_metrics_
 
 Tests can be aborted using CTRL-C but note that containers maybe still be running and will have to be shut down.
 
-#### Go
+### Using Pytest
+
+The tests are executed using pytest. Below are some common command-line options you can use to control and customize your test runs.
+- `-k EXPRESSION`: Run tests that match the given expression (substring or pattern). Useful for running specific tests or groups of tests.
+
+```sh
+TEST_LIBRARY=dotnet ./run.sh PARAMETRIC -k test_metrics_msgpack_serialization_TS001
```

Comment:
just a comment FYI, you can handle it in this PR, or in a later PR, or never :) 

The usual way to specify the lang is by using the `-L` argument : 

```suggestion
./run.sh PARAMETRIC -L dotnet -k test_metrics_msgpack_serialization_TS001
```

It's faster to type. The env var works fine but is longer to type. It;s commonly used for someone who **always** use only one library., and set this in it's `.env` file.